basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/ParseStringNodeType.java

```diff
@@ -0,0 +1,40 @@
+package club.mondaylunch.gatos.basicnodes;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import club.mondaylunch.gatos.core.data.DataBox;
+import club.mondaylunch.gatos.core.data.DataType;
+import club.mondaylunch.gatos.core.graph.connector.NodeConnector;
+import club.mondaylunch.gatos.core.graph.type.NodeType;
+
+public class ParseStringNodeType extends NodeType.Process {
+    @Override
+    public Map<String, DataBox<?>> settings() {
+        return Map.of();
+    }
+
+    @Override
+    public Set<NodeConnector.Input<?>> inputs(UUID nodeId, Map<String, DataBox<?>> state) {
+        return Set.of(
+            new NodeConnector.Input<>(nodeId, "input", DataType.STRING));
+    }
+
+    @Override
+    public Set<NodeConnector.Output<?>> outputs(UUID nodeId, Map<String, DataBox<?>> state) {
+        return Set.of(
+            new NodeConnector.Output<>(nodeId, "output", DataType.NUMBER));
+    }
+
+    @Override
+    public Map<String, CompletableFuture<DataBox<?>>> compute(Map<String, DataBox<?>> inputs, Map<String, DataBox<?>> settings) {
+        return Map.of(
+            "output", CompletableFuture.completedFuture(DataType.NUMBER.create(
+                Double.parseDouble(
```

Comment:
This can throw a NumberFormatException. Either catch this and return a value like 0 or NaN, or make the output an Optional and return an empty.